Touchmaster Delta Siren

Officer-Pringle opened the pull request:
Added a new Touchmaster Delta Siren,
This incudes a slightly louder .WAV and also a "jingle" Type of Phaser siren

- Horn - Schmal - Mastercom B
- Manual - Schmal - Touchmaster Delta

Officer-Pringle committed:
Touchmaster Delta Siren

Added a new Touchmaster Delta Siren,
This incudes a slightly louder .WAV and also a "jingle" Type of Phaser siren

JoshPiper reviewed (dismissed):
The yelp siren is currently clipping. That needs to be resolved before I can approve this.

SuperMighty1 commented:
Is it possible to merge this with the current TM Delta siren to include the phaser before the scan tone as that seems to be the only significant difference between them?

Officer-Pringle commented:
Would it be fine to do what SM Said and just add the phaser to the photon version of the Touchmaster Delta

Officer-Pringle committed:
Addition to Touchmaster Delta Siren

This is a addition to the main photon version that adds a "Phaser" Mode before the Scan Mode of the Original TMD Siren
-- Suggested to do this way by SM

JoshPiper commented on lua/autorun/photon/library/emv_sirens.lua:
```diff
@@ -36,6 +36,7 @@ local sirenTable = {
 		Set = {
 			{Name = "WAIL", Sound = "emv/sirens/federal sig tm/emv_wail.wav", Icon="wail"},
 			{Name = "YELP", Sound = "emv/sirens/federal sig tm/emv_yelp.wav", Icon="yelp"},
+			{Name = "PHSR", Sound = "sound/emv/sirens/p_fs_touchmaster_delta/pemv_tmjingle.wav", Icon = "phaser"} -- Edit by Pringle
```
This is missing a comma.

JoshPiper committed:
Update emv_sirens.lua

JoshPiper committed:
Don't prefix emv with sound/

JoshPiper committed:
It's a phaser that goes hi-lo.
